ramp-frontend/ramp_frontend/views/admin.py

```diff
@@ -312,7 +312,9 @@ def dashboard_submissions(event_name):
     name_submissions = [sub.name for sub in submissions]
     cumulated_submissions = list(range(1, 1 + len(submissions)))
     training_sec = [
-        (sub.training_timestamp - sub.submission_timestamp).seconds / 60.
```

Comment:
timedelta.seconds shows the number seconds modulo 1 day, and `total_seconds()` should be used to get the total amount of seconds.